hardware_interface/src/components/sensor.cpp

```diff
@@ -12,63 +12,46 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+#include <memory>
 #include <string>
+#include <utility>
 #include <vector>
 
-#include "hardware_interface/components/component_info.hpp"
 #include "hardware_interface/components/sensor.hpp"
-#include "hardware_interface/types/hardware_interface_return_values.hpp"
 
-#include "./component_lists_management.hpp"
+#include "hardware_interface/hardware_info.hpp"
+#include "hardware_interface/components/sensor_interface.hpp"
+#include "hardware_interface/types/hardware_interface_return_values.hpp"
+#include "hardware_interface/types/hardware_interface_status_values.hpp"
+#include "hardware_interface/visibility_control.h"
```

Comment:
don't we only need this include where the macros are being used sir?

Reply:
I guess that's part of #199

Reply:
ok